### DISCUSSION TITLE
Error during writing output file

### DISCUSSION THREAD

**Discussion by Sushmitharao46** [xcompact3d/Incompact3d#174] [Q&A] [answered]
Hi,
I'm getting the following error sometimes when the code reaches the **time step=ioutput**. 
**Note: The following floating-point exceptions are signalling: IEEE_DIVIDE_BY_ZERO IEEE_UNDERFLOW_FLAG IEEE_DENORMAL
 ERROR in MPI_FILE_WRITE_ALL**
Can someone please comment why this error is showing up.
Thank you.

**Comment by mathrack** [comment]
Hello,

Are you using a modified version of the code ? What kind of flow are you simulating ? What is your input file ? What is the compiler used ? What is the MPI library ? What is the file producing the error ? Can you reproduce the error on another machine ?

Your message is very vague.

Regards,
Mathrack

**Comment by Sushmitharao46** [comment]
Yes I've modified some of the ALM part of the code. I've attached the input file. I've used intel 2019 compiler. Yes I'm getting this error in some of the cases. 
[input.txt](https://github.com/xcompact3d/Incompact3d/files/11360526/input.txt)
Thank you.

**Comment by Sushmitharao46** [comment]
And also while writing a restart file sometimes I'm getting the following error. 
![image](https://user-images.githubusercontent.com/111350044/235392434-17846d1b-85af-47b0-bce0-d278631480e3.png)
Thank You.

**Comment by mathrack** [comment, marked as answer]
Hello,

The information provided are incomplete, I can not help you.

The error appears "sometimes" and you are using a modified version of the code. If you can reproduce the error, and if you provide more information, maybe I can help. Please follow this [link](https://github.com/2decomp-fft/2decomp-fft/blob/contribute/Contribute.md#bug) for general comments you can use to report a bug.

Regards,
Mathrack
